packages/wrangler/templates/middleware/loader-sw.ts

```diff
@@ -183,6 +183,8 @@ __facade__originalAddEventListener__("fetch", (event) => {
 			});
 
 			__FACADE_EVENT_TARGET__.dispatchEvent(facadeEvent);
+			// @ts-expect-error `waitUntil` types are currently broken, blocked on
+			// https://github.com/cloudflare/workerd/pull/191
```

Comment:
Can you make a wrangler issue to track fixing this once the workerd PR is merged?